### PR TITLE
Fix awkward clip animation on frame variables/http interface/others

### PR DESCRIPTION
**Pull request by benvinegar**
The problem was that items began in an unclipped state, then when they were added to the DOM, they became "clipped" and the animation began going from max-height 50000px -> 200px.

This way they begin in a clipped state first.

/cc @getsentry/ui @ckj

### CODE REVIEW

**Comment by ckj** [comment]
Ah, makes sense. Glad it was an easy fix 👍

File: src/sentry/static/sentry/app/components/clippedBox.jsx [hunk 1]
```diff
@@ -48,18 +51,24 @@ const ClippedBox = React.createClass({
...
           <h5>{this.props.title}</h5>
         }
         {this.props.children}
-        <div className="clip-fade">
-          <a onClick={this.reveal} className="show-more btn btn-primary btn-xs">
-            {t('Show more')}
-          </a>
-        </div>
+
+        {this.state.clipped &&
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/clippedBox.jsx]: Small optimization – it's better not to inject this DOM node if we don't need it

**Comment by ckj** [comment]
Looks good on my end. LGTM

**Comment by benvinegar** [comment]
Percy is also not as mad, so merging.

**Pull request merged**